.github/workflows/rebuildDependencies.yml

```diff
@@ -0,0 +1,78 @@
+name: VCMI - vcpkg dependencies
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 2 * * 0' #every sunday 02:00
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - platform: msvc-x64
+            triplet: x64-windows
+          - platform: msvc-x86
+            triplet: x86-windows
+
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout vcpkg repository
+      uses: actions/checkout@v4
+      with:
+        repository: 'microsoft/vcpkg'
+
+    # NOTE: disabling debug for both 32 and 64 bit triplets, so 64 host / 32 guest won't build debug version of 64-bit tools
+    - name: Disable debug builds
+      run: |
+        echo 'set(VCPKG_BUILD_TYPE release)' >>triplets/x64-windows.cmake
+        echo 'set(VCPKG_BUILD_TYPE release)' >>triplets/x86-windows.cmake
```

Comment:
So devs should build debug on their own? Or we care only about RelWithDebInfo local builds?

Reply:
1. Our debug configuration is broken for years, all devs use RelWithDebugInfo configuration
2. Full debug builds (including debug libraries) are *extremely* slow on msvc to a point of being unusable for games. Even VCMI would run at like <10 FPS. Better approach would be to build release libraries, use release runtime, but build vcmi code in debug configuration with no optimizations.